### DISCUSSION TITLE
Individual cell render into a table down below on onclick event

### DISCUSSION THREAD

**Discussion by sandeep1407** [KevinVandy/material-react-table#624] [Show and tell]
Hi, 
I am not sure that it is support to expand individual cell on onclick event similar to [Detail Panel (Expanding)](https://www.material-react-table.com/docs/guides/detail-panel). 

I have seen the example in another grid as below here.
[individual-cell-expand.docx](https://github.com/KevinVandy/material-react-table/files/12045102/individual-cell-expand.docx)

**Comment by KevinVandy** [comment]
You can easily implement that in MRT. You've got a lot of options actually. Custom `Cell` render in the column defs, hook up events in `muiTableBodyCellProps`, etc.

https://www.material-react-table.com/docs/guides/table-event-listeners

**Reply by sandeep1407** [reply]
Thank you for the reply Kevin. 
I have tried to use these option but unable to achieve it. I have written the demo code here in [codeSandbox](https://codesandbox.io/s/mystifying-noyce-z5mw3g?file=/src/Demo.js). 

Can you please help to make a link on individual cell and render into a Box or another table down in CodeSandbox.
Thank you so much for quick help.

**Comment by sandeep1407** [comment]
did you get a chance to look out this [codeSandbox](https://codesandbox.io/s/mystifying-noyce-z5mw3g?file=/src/Demo.js) please. I tried but its not rendering in place similar to [Detail Panel (Expanding)](https://www.material-react-table.com/docs/guides/detail-panel).

Thank you in advance.
